.github/workflows/release.yml

```diff
@@ -15,7 +15,22 @@ jobs:
       - uses: coursier/cache-action@v6.3
       - name: Publish
         run: |
-          sbt gh-actions-release
+          COMMAND="ci-release"
```

Comment:
We will be getting the `3.1.1` release soon, it will be perfect time to test it out!

Reply:
Tested this code locally with `CI_RELEASE=mtags/publishLocal` and I hope there will be no issues with bash code.

I'm already testing it on `3.1.2-RC1-bin-20220119-2849aed-NIGHTLY`, so could you put an approve? :wink: